src/ui/App.css

```diff
@@ -107,11 +107,13 @@ select:focus {
   width: 100%;
   height: 160px;
 
-  backdrop-filter: blur(10px);
-  box-shadow: inset 0px 5px 12px -3px rgb(50 50 50 / 75%);
-
   margin: 0;
-  padding: 0;
+  text-align: right;
+
+  /* Gradient shadow */
+  -webkit-box-shadow: inset 0px -150px 96px -85px rgba(0,0,0,0.43);
```

Comment:
First: The web engine will always be the same, so compat like `--webkit-*` is not needed

Second: I think we would prefer to keep the old backdrop blur, else the background makes some of the options busy and tough to see, *especially if the window is resized*. See screenshots:

Already feels busy because of the complexity of the background art:
![image](https://github.com/Grasscutters/Cultivation/assets/25207995/6de96e1e-e978-4ce5-9abc-85e483ec4b2e)

And when resized:
![image](https://github.com/Grasscutters/Cultivation/assets/25207995/eb19ca59-be1c-4b01-9215-7c90c2c17209)

Another problem is that the right bar now randomly cuts off:
![image](https://github.com/Grasscutters/Cultivation/assets/25207995/57569aaa-507d-418f-a5ff-fb3c83e10106)

So overall, just keep the old `backdrop-blur` please 👍

Reply:
I didn't take responsiveness into account since the original Genshin launcher isn't resizable and I noticed that responsiveness breaks anyway at some point so I just figured it's alright if it's kept this way. I'm just explaining my thought process, will revert back to the original design and take care of the other mistakes you pointed out. Thank you Spike!